docker_test.go

```diff
@@ -361,6 +423,86 @@ func run(m *testing.M) int {
 	webhookurl = webhook.Server.URL
 	fmt.Println("webhookurl", webhookurl)
 
+
+	// Setup MINIO
+	var minioClient *minio.Client
+
+	options := &dockertest.RunOptions{
+		Repository: "minio/minio",
+		Tag:        "latest",
+		Cmd:        []string{"server", "/data"},
+		PortBindings: map[dc.Port][]dc.PortBinding{
+			"9000/tcp": []dc.PortBinding{{HostPort: "9000"}},
```

Comment:
In the final PR, we should not bind to a specific external port, only a random port should be used.
```suggestion
			"9000/tcp": []dc.PortBinding{{HostPort: "9000"}},
```